recipes-app/iot2050-firmware-update/files/iot2050-firmware-update

```diff
@@ -289,13 +316,14 @@ class ManagedFirmwareUpdate(FirmwareUpdate):
         self._board_info = BoardInformation()
 
         # Parse the update configuration from the json file within the tarball
-        self._update_conf = UpdateConfiguration(
-            tarfile.open(fileobj=self.firmware).extractfile(self._update_conf_json))
+        self._update_conf = UpdateConfiguration(tarfile.open(
+            fileobj = self.firmware).extractfile(self._update_conf_json))
```

Comment:
IIRC, `=` in arguments should not have spaces around them, I might be wrong, you can do your own research.

Reply:
has removed the redundant blank